app/views/manage_consents/triage.html.erb

```diff
@@ -17,11 +17,28 @@
     <%= page_title %>
   <% end %>
 
-  <%= f.govuk_collection_radio_buttons :status,
-                                       triage_form_status_options,
-                                       :first,
-                                       :second,
-                                       legend: { text: "Outcome" } %>
+  <%= f.govuk_radio_buttons_fieldset(
+        :status, legend: { size: "m", text: "Outcome" },
+      ) do %>
+    <%= f.govuk_radio_button(
+          :status, :ready_to_vaccinate,
+          label: { text: Triage.human_enum_name(:status, :ready_to_vaccinate) },
+          link_errors: true,
+        ) %>
+    <%= f.govuk_radio_divider %>
+    <%= f.govuk_radio_button(
+          :status, :do_not_vaccinate,
+          label: { text: Triage.human_enum_name(:status, :do_not_vaccinate) },
+        ) %>
+    <%= f.govuk_radio_button(
+          :status, :delay_vaccination,
+          label: { text: Triage.human_enum_name(:status, :delay_vaccination) },
+        ) %>
+    <%= f.govuk_radio_button(
+          :status, :needs_follow_up,
+          label: { text: Triage.human_enum_name(:status, :needs_follow_up) },
+        ) %>
+  <% end %>
```

Comment:
This looks the same to me as the fields in `app_triage_form_component.html.erb`, is there a reason we wouldn't use that component or can we extract this to somewhere shared across both the component and this view?

Reply:
Yup, but I felt like the code looked uglier/harder to understand than without...

Here's something similar to what I tried:

```erb
<%= f.govuk_radio_button(*radio_options(:ready_to_vaccinate)) %>
<%= f.govuk_radio_divider %>
<%= f.govuk_radio_button(*radio_options(: do_not_vaccinate)) %>
<%= f.govuk_radio_button(*radio_options(: delay_vaccination)) %>
<%= f.govuk_radio_button(*radio_options(: needs_follow_up)) %>
```

Where `radio_options` is a helper that returns out `[:status, :ready_to_vaccinate, { label: ... }]`.

However, Ruby is weird about how it handles splatting and keyword args. The code above would error due to passing in 3 arguments when 2 are required. This is because the final argument in my `radio_buttons` helper is a hash, but the radio expects kwargs, which are subtly different. The hash argument needs a double splat `**` but there's no easy way to do that.

So this would work, but it's really ugly:

```erb
<%= f.govuk_radio_button(*radio_options(:ready_to_vaccinate)[0..2], **radio_options(:ready_to_vaccinate).last) %>
```

After 30 minutes spent trying to golf it I decided the duplication isn't so bad yet...

If you can suggest a different way, happy to try it 👍

Reply:
Ah, the reason not to use the component is because the forms are different I guess. I didn't really consider trying to make `AppTriageFormComponent` fit this usecase. I'll look into that now.

Reply:
I've pushed up a commit, I massaged the `AppTriageFormComponent` to make it more flexible to fit this usecase. 💪